mne/chpi.py

```diff
@@ -678,17 +685,19 @@ def _fit_chpi_amplitudes(raw, time_sl, hpi):
         if not (n_on >= 3).all():
             return None
     return _fast_fit(this_data, hpi['proj_op'], len(hpi['freqs']),
-                     hpi['model'], hpi['inv_model_reord'])
+                     hpi['model'], hpi['inv_model'], hpi['inv_model_reord'],
+                     hpi['mag_picks'], hpi['grad_picks'], return_snr)
 
 
-@jit()
```

Comment:
We definitely want this back in there eventually, optimizing these is important. To avoid making numba angry (which is probably due to differing `return` statements), you can make this param `compute_snr` and if it's False, just return all-zero arrays of the correct size.

Reply:
that was an accidental omission, I meant to put it back.  (removed it while debugging)